process/OpenHW-Specification-Process.md

```diff
@@ -0,0 +1,170 @@
+# OpenHW Process Document: Specification Process, States and Format
+
+## Revision History
+
+This document desbribes OpenHW's specification process and format.
+The overall approach was adopted by consensus in the TWG meeting of July 2023 based on power point presentations and other documents.
+
+The V1.0 of this document reflects the agreements made at that meeting and adds additional details. 
+
+| Date        | Version | Notes                                                     |
+| :---------- | :------ | :-------------------------------------------------------- |
+| 22 Jan 2024 | 1.0     | Formal draft reflecting July 2023 TWG meeting materials	|
+                                  
+## Specification Process
+
+### Overview
+
+OpenHW uses a lightweight specification development process to produce specification documents that are developed and frozen in a designated Task Group, then ratified and released by TWG.
+
+### Requirements for OpenHW Specification Process
```

Comment:
I do not understand how this document can be used by contributors to specifications. Further, I see "requirements", but no text is written as a requirement, typical with the "shall" keyword.

Reply:
New intro is OK.